ChangeLog.txt

```diff
@@ -1,3 +1,21 @@
+ParseCSV 1.0.0 RC

```

Comment:
I'd like to make the case for officially using [Semantic Versioning](https://semver.org/). Meaning this line would read:

    ParseCSV 1.0.0-rc.1